pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -51,6 +51,7 @@
   private static final String JERSEY_ADMIN_API_PORT = "jersey.admin.api.port";
   private static final String JERSEY_ADMIN_IS_PRIMARY = "jersey.admin.isprimary";
   private static final String ACCESS_CONTROL_FACTORY_CLASS = "controller.admin.access.control.factory.class";
+  private static final String SEGMENT_PUSH_TIMEOUT_MILLIS = "segment.push.timeout.millis";
```

Comment:
will be good to add a comment here on what this config means. Also, suggest rename to "segment.commit.timeout.millis"